plugins/kubernetes/test/models/kubernetes/template_filler_test.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require_relative "../../test_helper"
 
-SingleCov.covered! uncovered: 3
+SingleCov.covered! uncovered: 4
```

Comment:
this number is supposed to go down :D

Reply:
Thought it was a bug like we saw before but I found the branch that wasn't covered.